.github/actions/install-ngc-cli/action.yml

```diff
@@ -0,0 +1,20 @@
+name: install-ngc-cli
```

Comment:
Since this is now its own action, it would be nice to update docker_images.yml and publishing.yml to use it.

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1227/commits/2da783c3988a680fef5d7f6aa360da33fe7a33a5